scripts/upload_ci_stats.sh

```diff
@@ -29,7 +29,13 @@ create table job(
     url text,
     run_attempt int,
     run_id bigint,
-    run_number int
+    run_number int,
+    pg_version text generated always as (
```

Comment:
I'd write just `(regexp_match(job_name, '.*PG([^ ]+).*'))[1]`
And this doesn't work for the fuzzing jobs, but probably that's not important.

Reply:
> I'd write just `(regexp_match(job_name, '.*PG([^ ]+).*'))[1]`

I wanted to have something closer to a real version string so that we do not have too many special-cased searches in the future, but yeah, this can be simplified.

>And this doesn't work for the fuzzing jobs, but probably that's not important.

Yeah, the column can be null, so you can easily filter those out. Also, if we decide to add the version to the job names, it will automatically show up.

Reply:
Simplified.